features/layout/card-blocks-layout.js

```diff
@@ -12,16 +12,25 @@ import {
   TagsById,
 } from 'utils/customPropTypes';
 
-const CardBlocksLayout = ({ blocks, data }) => {
+const CardBlocksLayout = ({ blocks, data, inViewport }) => {
   const blocksByType = keyBy(blocks, 'type');
   return blocks.map((block, index) => {
     const Block = BLOCKS_BY_TYPE[block.type];
     if (!Block) {
       return null;
     }
+
+    const maxViewportBlockNumber = block.type === 'banner' ? 2 : 1;
```

Comment:
banners are not the only "short" blocks. row of three is pretty short too. so is diary. 

sample with 3 blocks in viewport:
<img width="1767" alt="Screen Shot 2020-12-29 at 22 03 24" src="https://user-images.githubusercontent.com/7117196/103307686-bff7db00-4a21-11eb-8d2a-a8f6e2c4ea9e.png">

let's not make it *that* optimized. let's just make maxViewportBlockNumber constant as 3 or 4.